heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/DryRunRender.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.scheduler.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.twitter.heron.spi.packing.PackingPlan;
+import com.twitter.heron.spi.packing.Resource;
+
+/**
+ * Interface of class that renders dry-run response
+ */
+public abstract class DryRunRender {
+  protected Map<String, Resource> componentsResource(PackingPlan packingPlan) {
```

Comment:
Parallelism mapping isn't needed, it's already done in `packingPlan.getComponentCounts()`. If we need to sum all resources by component, we should also provide that as a method in `PackingPlan`. And it should have a unit test.